tests/unit/src/main/scala/tests/BaseExpectSuite.scala

```diff
@@ -20,7 +20,8 @@ abstract class BaseExpectSuite(val suiteName: String) extends BaseSuite {
         }
         .getOrElse(Classpath(Nil))
     GlobalSymbolTable(
-      input.classpath ++ bootClasspath
+      input.classpath ++ bootClasspath,
```

Comment:
Note for myself: the `GlobalSymbolTable` uses `ClasspathIndex` under the hood, so manually adding in the bootClasspath isn't needed, as it already is applied by the `includeJdk = true`: https://github.com/scalameta/scalameta/blob/18229e37d89f7982ef65a8b034edb0f503fd63cc/semanticdb/metacp/src/main/scala/scala/meta/internal/classpath/ClasspathIndex.scala#L67-L82

Try deleting the (duplicated) bootClasspath code here and make sure it doesn't affect anything, also saw: https://github.com/scalameta/metals/pull/767/files which seemed to strip it out without any undesirable side effects.